components/org.wso2.carbon.identity.conditional.auth.functions.client.assertion.parameters/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/jwt/decode/ClientAssertionParametersImpl.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.wso2.carbon.identity.conditional.auth.functions.client.assertion.parameters;
+
+import com.nimbusds.jose.JWSObject;
+import net.minidev.json.JSONObject;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.authentication.framework.exception.FrameworkException;
+
+import java.text.ParseException;
+
+/**
+ * Represents javascript function provided in conditional authentication to decode a jwt assertion and retrieve
+ * a particular value in the assertion
+ */
+public class ClientAssertionParametersImpl implements ClientAssertionParameters {
```

Comment:
Do we need this? `JwtDecodeImpl` is also similar to this.

Reply:
removed file